cloud-stream-batch-consume/src/main/resources/application.yaml

```diff
@@ -0,0 +1,28 @@
+spring:
+  cloud:
+    function:
+      definition: batchConsume
+    stream:
+      bindings:
+        batchConsume-in-0:
+          destination: 'batch/consume/topic'
+          group: batch
+          content-type: 'application/octet-stream'
+          consumer:
+            batch-mode: true
+            useNativeDecoding: true
+      binders:
+        solace-broker:
+          type: solace
+          environment:
+            solace: # (1)
+              java:
+                host: tcp://localhost:55555
+                msgVpn: default
+                clientUsername: default
+                clientPassword: default
+                connectRetries: -1
+                reconnectRetries: -1
+server:
+  port: 8097
```

Comment:
i believe you removed thie from the other samples

Reply:
Did the removal on other yaml files as well.

Reply:
ty